test/evm/hardhat/SpokePool.Deposit.ts

```diff
@@ -407,40 +390,44 @@ describe("SpokePool Depositor Logic", async function () {
       depositArgs = getDepositArgsFromRelayData(relayData);
     });
     it("placeholder: gas test", async function () {
-      await spokePool.connect(depositor)[depositV3Bytes](...depositArgs);
+      await spokePool.connect(depositor)[SpokePoolFuncs.depositV3Bytes](...depositArgs);
```

Comment:
Hardhat does not automatically pick overloaded functions, so we need to explicitly specify the version.